jobs/transforms/skill_validation.py

```diff
@@ -58,8 +60,10 @@ class ValidateSkillCommitLogEntryModel(
         base_validation.BaseValidateCommitCmdsSchema):
     """Overrides _get_change_domain_class for SkillCommitLogEntryModel."""
 
-    def _get_change_domain_class(self, input_model): # pylint: disable=unused-argument
-        # type: (Any) -> Optional[Type[skill_domain.SkillChange]]
+    def _get_change_domain_class(
+            self,
+            input_model: Any
```

Comment:
Why `Any`?

Reply:
Removed this file now - see [this](https://github.com/oppia/oppia/pull/13681#discussion_r690241356) comment.